rules/windows/process_creation/proc_creation_win_lolbin_manage_bde.yml

```
@@ -0,0 +1,36 @@
+title: Potential Manage-bde.wsf Abuse To Proxy Execution
+id: c363385c-f75d-4753-a108-c1a8e28bdbda
+status: test
+description: Detects potential abuse of the "manage-bde.wsf" script as a LOLBIN to proxy execution
+references:
+    - https://lolbas-project.github.io/lolbas/Scripts/Manage-bde/
+    - https://gist.github.com/bohops/735edb7494fe1bd1010d67823842b712
+    - https://twitter.com/bohops/status/980659399495741441
+    - https://twitter.com/JohnLaTwC/status/1223292479270600706
+    - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1216/T1216.md
+author: oscd.community, Natalia Shornikova, Nasreddine Bencherchali (Nextron Systems)
+date: 2020/10/13
+modified: 2023/02/03
+tags:
+    - attack.defense_evasion
+    - attack.t1216
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_wscript_img:
+        - Image|endswith: '\wscript.exe'
+        - OriginalFileName: 'wscript.exe'
+    selection_wscript_cli:
+        CommandLine|contains: 'manage-bde.wsf'
+    selection_parent:
+        ParentImage|endswith:
+            - '\cscript.exe'
+            - '\wscript.exe'
+        ParentCommandLine|contains: 'manage-bde.wsf'
+    filter_cmd:
```

```suggestion
    filter_sel_parent:
```
My idea is to have a name that makes it clear that it is not a global filter but for selection_parent.

---

I will use `selection_filter_cmd` and we will use `selection_filter_` notation for these kind of filters that are not FP